packages/core/src/JWEditor.ts

```diff
@@ -419,6 +419,12 @@ export class JWEditor {
                     throw error;
                 }
                 console.error(error);
+
+                await this.dispatcher.dispatchHooks('@error', {
+                    message: error.message,
+                    stack: error.stack,
+                });
+
```

Comment:
I don't understand the use of this. Isn't console.error sufficient ?

Reply:
Si c'est juste pour l'avoir dans le log. En fait si tu continues à éditeur ou qu'il y a plusieurs commandes, ça aide de savoir quand l'erreur s'en produite.